### PR TITLE
fix: Add Apphub URI output to Redis

### PR DESCRIPTION
**Pull request by tjy9206**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -68,3 +68,11 @@ output "env_vars" {
     "REDIS_PORT" : tostring(google_redis_instance.default.port)
   }
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//memorystore.googleapis.com/${google_redis_instance.default.id}"
```
Comment (q2w) [inline outputs.tf:74]:
```suggestion
    service_uri = "//redis.googleapis.com/${google_redis_instance.default.id}"
```

File: outputs.tf [hunk 2]
```diff
@@ -68,3 +68,11 @@ output "env_vars" {
...
+    service_id  = google_redis_instance.default.id
```
Comment (q2w) [inline outputs.tf:75]:
```suggestion
    service_id  = substr("${var.name}-${md5("${var.region}-${var.project_id}")}", 0, 63)
```

**Review by q2w — APPROVED** [review]

**Review by apeabody — COMMENTED** [review]
As this PR modifies a `.tf` file, I would recommend calling as a "fix".  That way it will included it in the release notes, and more importantly, if one doesn't exist, triggers the creation of a new release PR ("chore" do not).

**Pull request merged**